lib/index.js

```diff
@@ -86,22 +87,27 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
+  var opts = this.options;
+  var eventProperties = track.properties();
   // Use the new-style API (which is compatible with Classic and X)
   var payload = {
     type: 'event',
     eventName: track.event(),
     tags: track.properties()
```

Comment:
@ladanazita i think you forgot to update this line here to `tags: eventProperties`